packages/suite/src/components/suite/OverflowItems.tsx

```diff
@@ -0,0 +1,134 @@
+import { ElementType, ReactNode, useCallback, useEffect, useRef, useState } from 'react';
+import { usePrevious, useShallowCompareEffect, useUpdateEffect } from 'react-use';
+import { useSelector } from 'src/hooks/suite';
+
+type Item = {
+    removeOrder?: number;
+    [key: string]: any;
+};
+
+interface OverflowItemsProps<T> {
+    wrapper?: ElementType;
+    items: T[];
+    visibleItemRenderer: (item: T, index: number) => ReactNode;
+    overflowRenderer: (items: T[]) => ReactNode;
+    minVisibleItems?: number;
+    alwaysRenderOverflow?: boolean;
+}
+
+export function OverflowItems<T extends Item>(props: OverflowItemsProps<T>) {
```

Comment:
this behavior exists in [AppNavigation](https://github.com/trezor/trezor-suite/blob/89834958042c4e046a0d8faf4e316b3ca37066e7/packages/suite/src/components/suite/AppNavigation.tsx#L211) as well

Reply:
not really, in AppNavigation all buttons are hidden if one of them doesn't fit, here we want to gradually hide buttons one by one

Reply:
`OverflowItems` seems like improved behavior to me + we would avoid logic/code duplication. Looks like an improvement to me, wdyt? @matejkriz @Hermez-cz